README.md

```diff
@@ -8,15 +8,18 @@ SPDX-License-Identifier: Apache-2.0
 # Welcome to TieDie IoT
 
 This package enables IoT device provisioning and communication for
-non-IP devices (e.g., BLE, Zigbee) via an application layer gateway.
-Want to deploy a new application?  You no longer need to lay down
-additional bridges or make use of USB dongles.
+non-IP devices such as [Bluetooth Low
+Energy](https://www.bluetooth.com/specifications/specs/core-specification-5-4/)
+and [Zigbee Z-Wave](https://sdomembers.z-wavealliance.org/document/dl/917) via an
```

Comment:
Zigbee is different from z-wave and is from the CSA, not the z-wave alliance.

Reply:
corrected.